services/catalog/service/src/graphql/plugins/entity-localization-plugin.ts

```diff
@@ -0,0 +1,83 @@
+import { Plugin } from 'graphile-build';
+import { QueryBuilder } from 'graphile-build-pg';
+import {
+  embed,
+  gql,
+  makeExtendSchemaPlugin,
+  makePluginByCombiningPlugins,
+} from 'graphile-utils';
+
+import { Table } from 'zapatos/schema';
+import { MOSAIC_LOCALE_PG_KEY } from '../../common';
+
+const wellKnownProperties = [
```

Comment:
Instead of an array you may use an object then you do not need "find" to search through the array but could access it via `wellKnownProperties[property]`